src/BuildingRegistry.Projector/Infrastructure/Modules/ApiModule.cs

```diff
@@ -96,6 +98,30 @@ private void RegisterProjectionSetup(ContainerBuilder builder)
             }
         }
 
+        private void RegisterAddressLinkExtractV2Projections(ContainerBuilder builder)
+        {
+            builder.RegisterModule(
+                new ExtractModule(
+                    _configuration,
+                    _services,
+                    _loggerFactory));
+
+            builder
+                .RegisterProjectionMigrator<ExtractContextMigrationFactory>(
+                    _configuration,
+                    _loggerFactory)
+                .RegisterProjections<BuildingUnitAddressLinkExtractProjections, ExtractContext>(
+                    context => new BuildingUnitAddressLinkExtractProjections(
+                        context.Resolve<IOptions<ExtractConfig>>(),
+                        DbaseCodePage.Western_European_ANSI.ToEncoding()),
+                    ConnectedProjectionSettings.Default)
+                .RegisterProjections<BuildingUnitAddressLinkExtractProjections, ExtractContext>(
+                    context => new BuildingUnitAddressLinkExtractProjections(
+                        context.Resolve<IOptions<ExtractConfig>>(),
+                        DbaseCodePage.Western_European_ANSI.ToEncoding()),
+                    ConnectedProjectionSettings.Default);
```

Comment:
It's registered twice. Can you add it to `RegisterExtractProjectionsV2` instead?